src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertGetCertificateChain.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypt32
+    {
+        [GeneratedDllImport(Libraries.Crypt32, SetLastError = true)]
+        internal static unsafe partial bool CertGetCertificateChain(IntPtr hChainEngine,
```

Comment:
Personally, I'd put all of the contents of CERT_CHAIN_PARA.cs into this file, since there's no need for it with any other function.